app/views/admin/campaigns/index.html.haml

```diff
@@ -18,16 +18,16 @@
       %tbody
         - @campaigns.each do |campaign|
           %tr
-            %td{'id'=> "name_#{campaign.id}"}
+            %td{ 'id' => "name_#{campaign.id}" }
```

Comment:
Why not `%td{ id: "name_#{campaign.id}" }`

This applies to the entire PR, not just this line.